core/interfaces.go

```diff
@@ -1078,6 +1078,8 @@ type RokmetroJoinGroupExtApprovement struct {
 	GroupName                string    `json:"group_name"`
 	FirstName                string    `json:"first_name"`
 	LastName                 string    `json:"last_name"`
+	Email                    string    `json:"email"`
```

Comment:
It is better to use a pointer *string for phone and email because when the field data is not presented then it should be null instead of "" (empty string).